packages/transport/src/core/Reference.ts

```diff
@@ -9,9 +9,12 @@ export interface IReference extends ISerializable {
     backboneBaseUrl?: string;
     key: ICryptoSecretKey;
     forIdentityTruncated?: string;
-    passwordType?: number;
+    passwordType?: string;
+    version?: number;
```

Comment:
what do we use this version for? I don't get it.

Reply:
Giving a version to the reference was for easier updating of the reference (originally for if the password algorithm changes, but we decided to apply the version to the entire reference instead of just the password)

Reply:
This feels so unrelated to the other changes here, this should defenitely not be made in this PR.